contrib/database/sql/stmt.go

```diff
@@ -35,7 +35,7 @@ func (s *tracedStmt) ExecContext(ctx context.Context, args []driver.NamedValue)
 	start := time.Now()
 	if stmtExecContext, ok := s.Stmt.(driver.StmtExecContext); ok {
 		res, err := stmtExecContext.ExecContext(ctx, args)
-		s.tryTrace(ctx, queryTypeExec, s.query, start, err)
+		s.tryTrace(ctx, queryTypeExec, s.query, start, err, 0)
```

Comment:
Ok, I see 🙂 That's why your variadic version was better. I was not aware of this side-effect when I commented to change that.

Reply:
I reverted to the previous version.